pipelineScript.py

```diff
@@ -136,13 +143,51 @@ def get_log_items(pipeline_path):
         offset = -1
 
     # Convert objects to json objects
+    #json_logs = []
+    #for i in range(len(logItems)):
+     #   json_logs.append(logItems[i].__dict__)
+    return logItems
+
+# Gets the key for a sorting algorithm
+# @params the LogItem to return the offset from
+# @return the offset of the logItem
+def get_offset(logItem):
+    return logItem.offset
+
+# Groups the logItem chunks together based on offset
+# @param the path to the pipeline log file
+# @return a list of the new merged chunks
+def get_chunks(path):
+    logItems = get_log_items(path)
+    logItems.sort(key=get_offset)
+
+    chunks = []
+    chunks.append(logItems[0])
+    tempCluster = logItems[0].clusterName
+    chunks[0].clusterName = []
+    chunks[0].clusterName.append(tempCluster)
+    index = 0
+    for i in range(1, len(logItems)):
```

Comment:
might be a slightly more idiomatic to:
`for logItem in logItems:`

Reply:
Since I need to compare each log Item with the previous log item, I chose to use the "for i in range" type of Loop. The "for logItem in logItems" would require a different approach.